README.md

```diff
@@ -6,39 +6,28 @@
  [![Maintainability](https://api.codeclimate.com/v1/badges/fd0577360749c9b3d166/maintainability)](https://codeclimate.com/github/GSA/search-gov/maintainability)
 
 ## Contributing to search-gov
-Read our [contributing guidelines](./CONTRIBUTING.md). 
+Read our [contributing guidelines](https://github.com/GSA/search-gov/blob/master/CONTRIBUTING.md). 
```

Comment:
Looks like the merge commit pulled in an unintended change. This should be a relative link:
```
Read our [contributing guidelines](./CONTRIBUTING.md). 
```